### PR TITLE
asadm 2.10.0

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
Key | Summary 
-- | -- 
TOOLS-2166 | (ASADM) The `summary` command does not show `F-` to indicate federal edition. |   | Actions
TOOLS-2148 | (ASADM) Running the `summary` command multiple times breaks the "OS Version" field |   |  
TOOLS-2146 | (ASADM) `info sindex` state reads "Read-Only" instead of "Read-Write`. |   |  
TOOLS-1950 | (ASADM) Add sudo ethtool -S command to collectinfo |   |  

TOOLS-2166	
[(ASADM) The `summary` command does not show `F-` to indicate federal edition.](https://aerospike.atlassian.net/browse/TOOLS-2166)

Bug	
[TOOLS-2148](https://aerospike.atlassian.net/browse/TOOLS-2148)	
[(ASADM) Running the `summary` command multiple times breaks the "OS Version" field](https://aerospike.atlassian.net/browse/TOOLS-2148)

Bug	
[TOOLS-2146](https://aerospike.atlassian.net/browse/TOOLS-2146)	
[(ASADM) `info sindex` state reads "Read-Only" instead of "Read-Write`.](https://aerospike.atlassian.net/browse/TOOLS-2146)

Bug	
[TOOLS-1950](https://aerospike.atlassian.net/browse/TOOLS-1950)	
[(ASADM) Add sudo ethtool -S command to collectinfo](https://aerospike.atlassian.net/browse/TOOLS-1950)

Improvement

### CODE REVIEW

**Comment by jdogmcsteezy** [comment]
Tests pass but fail on upload to codecov.  Looks like a reported issue

**Review by dwelch-spike — APPROVED** [review]
LGTM, lets wait till we get approval from QE to merge.

**Review by atefehmohseni — APPROVED** [review]
LGTM

**Pull request merged**
